src/main/java/academy/mate/bookstore/repository/impl/BookRepositoryImpl.java

```diff
@@ -32,7 +30,7 @@ public Book save(Book book) {
             if (transaction != null) {
                 transaction.rollback();
             }
-            throw new RuntimeException("Can`t save book to DB", e);
+            throw new EntityNotFoundException("Can`t create book " + book, e);
```

Comment:
EntityNotFoundException on dao layer is inappropriate. Please return back to RuntimeException